src/app/shared/common.scss

```diff
@@ -225,3 +225,7 @@ ul {
   overflow: hidden;
 }
 
+post-it-document.card:hover .icon {
+  opacity: 1 !important;
```

Comment:
We should place such selectors inside components. Or is there something that prevents us from doing so? Do we want to target all icons inside the child components of `PostItDocumentComponent`? And if so, cannot this be addressed directly in those components?

Reply:
We discussed this with @marvec in live coding two weeks ago. When we tried to place this selector inside component, it doesn't work.